www/includes/easyparliament/templates/html/mp/_vote_description.php

```diff
@@ -8,7 +8,7 @@
         <?= isset($party_voting_line) ? $party_voting_line : '' ?>
     </a>
     <?php if (isset($covid_affected) && $covid_affected) { ?>
-    <span style="font-size:60%">Absences for this policy may be affected <a href="#covid-19">COVID-19 restrictions</a>.</span>
+    <span style="font-size:60%">Absences for this policy may be affected <a href="votes#covid-19">COVID-19 restrictions</a>.</span>
```

Comment:
Ooh, I was going to say I don't think this URL will work, as main profile is /mp/ID/name/const so it would go to /mp/ID/name/votes - but that does appear to redirect to the right place. But it'd be better if it were `<?= $member_url ?>/votes` I think, to match other uses.